core/cap-0035.md

```diff
@@ -292,10 +292,9 @@ index 7f08d757..7476f1b6 100644
 +
 +    // codes considered as "failure" for the operation
 +    CLAWBACK_MALFORMED = -1,
-+    CLAWBACK_NOT_ISSUER = -2,
-+    CLAWBACK_NOT_CLAWBACK_ENABLED = -3,
-+    CLAWBACK_NO_TRUST = -4,
-+    CLAWBACK_UNDERFUNDED = -5
++    CLAWBACK_NOT_CLAWBACK_ENABLED = -2,
++    CLAWBACK_NO_TRUST = -3,
++    CLAWBACK_UNDERFUNDED = -4

```

Comment:
This change corrupted the diff. Because this change is an actual diff/patch, changing a single line can alter the rest of the diff. It's safer to do it in the stellar-core repo and regenerate the diff each time I've found.

Reply:
Ah my bad. I'm not too familiar with git diff and thought it would be fine. I'll make sure to do it right next time. I'll also open a PR to fix the diff Tuesday morning.

Reply:
No worries, I got you: https://github.com/stellar/stellar-protocol/pull/835.

Reply:
@MonsieurNicolas Now that we're moving to git diffs in CAPs since 14df315e40a77375e87ce88b9952af6b98aec8c4 was introduced, maybe we need some sort of automation on this repo to check that diffs in CAPs apply successfully to the version of Core they are intended to modify? We should be able to rig that up within a GitHub Action if we add a meta field to the CAP to specify which version of Core the CAP is a diff against.

Reply:
Bleh, I think we're talking about adding some meta data to md files (so that we can reliably find "patch blocks") and runtime dependency on a different repo. This may get complicated quickly. Alternative could be to move the diff to a different file that is just a patch file against XDR files (basically the output of `git diff -p` which allows for multiple files to be patched), to at least avoid the meta data problem.
I'd say we should give the manual process a shot (what we currently do), and add automated enforcement if we're running into problems too often. This should be tracked as a "meta" issue as we'd be changing the CAP process (and maybe somebody in the community could pick it up too, I think GA are javascript).